src/components/PendingTxsList/index.tsx

```diff
@@ -0,0 +1,139 @@
+import { TxInfo } from 'src/routes/safe/components/Transactions/TxList/TxCollapsed'
+
+import OutgoingTxIcon from 'src/routes/safe/components/Transactions/TxList/assets/outgoing.svg'
+import SettingsTxIcon from 'src/routes/safe/components/Transactions/TxList/assets/settings.svg'
+
+import styled from 'styled-components'
+import Spacer from 'src/components/Spacer'
+
+// Pending transactions data to be implemented BE endpoint
+import mockData from './mockData.json'
+type SafePendingTransactionsType = {
+  safeAddress: string
+  chainId: ChainId
+  totalPending: number
+  summaries: TransactionSummary[]
+}
+const mock = mockData as unknown as SafePendingTransactionsType[]
+
+// Displays up to 5 transactions across chains
+const transactionsToDisplay = (data: SafePendingTransactionsType[]): SafePendingTransactionsType[] => {
+  const MAX_PENDING_TXS = 5
+
+  // If there are more than 5 Safes with pending transactions
+  if (data.length >= MAX_PENDING_TXS) {
+    return data.slice(0, MAX_PENDING_TXS).map((safe) => ({ ...safe, summaries: [safe.summaries[0]] }))
+  }
+
+  // If there are less than MAX_PENDING_TXS in total
+  const totalPendingTransactions = data.reduce((acc, val) => acc + val.totalPending, 0)
+  if (totalPendingTransactions <= MAX_PENDING_TXS) {
+    return data
+  }
+
+  const safesToDisplay: SafePendingTransactionsType[] = []
+  let countDisplayTransactions = 0
+  let summariesIndex = 0
+
+  // Fill MAX_PENDING_TXS transactions to display
+  while (countDisplayTransactions < MAX_PENDING_TXS) {
+    for (let i = 0; i < data.length; i++) {
+      if (summariesIndex === 0) {
+        // Deep copy
+        const safe = JSON.parse(JSON.stringify(data[i]))
+        safe.summaries = [safe.summaries[summariesIndex]]
+        safesToDisplay.push(safe)
+      } else {
+        const nextTxSummary = data[i].summaries[summariesIndex]
+        nextTxSummary && safesToDisplay[i].summaries.push(nextTxSummary)
+      }
+      countDisplayTransactions++
+    }
+    summariesIndex++
+  }
+  return safesToDisplay
+}
+
+const PendingTxsList = (): ReactElement => {
+  return (
+    <>
+      <h2>Transactions to Sign</h2>
+      <List component="div">
+        {transactionsToDisplay(mock).map(({ safeAddress, chainId, totalPending, summaries }) => {
+          const { shortName } = getChainById(chainId)
+          const url = generateSafeRoute(SAFE_ROUTES.TRANSACTIONS_QUEUE, { safeAddress, shortName })
+          return summaries.map((transaction) => {
+            const info = getAssetInfo(transaction.txInfo)
+            return (
+              <TransactionToConfirm key={transaction.id} to={url}>
+                <OverlapDots>
+                  <CircleDot networkId={chainId} />
+                  <div style={{ width: `${lg}`, height: `${lg}` }}>
+                    <Identicon address={safeAddress} size="sm" />
+                  </div>
+                </OverlapDots>
+                <CustomIconText
+                  address="0x"
```

Comment:
Why send this prop at all? If it's necessary, we have a constant for this: `EMPTY_DATA`.